core/src/main/java/io/questdb/griffin/SqlCompiler.java

```diff
@@ -2016,7 +1969,9 @@ private long addTextImportRequest(CopyModel model, @Nullable CharSequence fileNa
                     throw SqlException.$(0, "Unable to process the import request. Another import request may be in progress.");
                 }
             } else {
-                throw SqlException.$(0, "Another import request is in progress.");
+                importIdSink.clear();
+                Numbers.appendHex(importIdSink, inProgressImportId, true);
+                throw SqlException.$(0, "Another import request is in progress. Import ID in progress = ").put(importIdSink);
```

Comment:
nit: the following formatting would be more consistent with other error messages
```suggestion
                throw SqlException.$(0, "Another import request is in progress ").put("[activeImportId=").put(importIdSink).put("]");
```